envoy.yaml

```diff
@@ -0,0 +1,61 @@
+admin:
```

Comment:
better you don't upload this envoy.yaml iz, you can delete this file

Reply:
why mba?

Reply:
`envoy.yaml` and `.env` files are better not published because they usually contain sensitive informations like credentials